.github/workflows/static-analysis.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         java-version: 11
         cache: gradle
     - name: Check for Detekt Issues
-      uses: burrunan/gradle-cache-action@v1
+      uses: gradle/gradle-build-action@v2@v1
```

Comment:
Please drop the `@1` here.

Reply:
Resolved in [`74109cb`](https://github.com/oss-review-toolkit/ort/pull/5749/commits/74109cbd039eb2cb8d42bf21586b76ac14d6a464)